doc/installation-guide/server.rst

```diff
@@ -34,25 +40,22 @@ Choose the following parameters:
 * OS Type : your Operating System
 * DB Type : "Mixed type of applications"
 * Total memory : how much memory is installed on your server
-* Number of connections : how many concurrent users you want to allow ( as a maximum ). Optional parameter
+* Number of connections : how many concurrent users you want to allow (as a maximum). Optional parameter
 
 The application should give you the settings to adapt in the ``postgresql.conf`` configuration file.
 
-Client-side
-===========
 
-Client side components are:
+.. note::
 
-* QGIS (2.14+)
-* QGIS extensions (depending on specific use case)
+ In a single desktop environment (no network database server) you can setup
+ your database to trust local connections.
 
-The exact required configuration is very dependant on the data sizes and complexity of rendering.
-However, water network data tend not to be huge volumes, and the minimal required configuration
-is not really high. For smooth map rendering, it is advised to use recent hardware, good video card and enough RAM though.
+ This way, the database won't ask for a password if you're connecting from your
+ local workstation while no password is written  anywhere. You even can do
```

Comment:
workstation vs machine

And you've got an extra space between "written" and "anywhere". (Might have no effect in RST, I am not sure.)